priv/static/themes/ui-protection.css

```
@@ -0,0 +1,458 @@
+[data-phx-component*="flash"][class*="rose"],
+[data-phx-component*="flash"][class*="red"],
+.flash.error,
+.alert.error,
+.phx-flash-error,
+div[phx-flash-kind="error"],
+p[role="alert"][class*="rose"],
+p[role="alert"][class*="red"] {
+  background-color: #fef2f2 !important; /* rose-50 */
+  color: #991b1b !important; /* rose-800 */
+  border-color: #f87171 !important; /* rose-400 */
+  border: 1px solid #f87171 !important;
```

Biome (1.9.4)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)